pkg/cmd/roachprod/vm/aws/support.go

```diff
@@ -56,6 +57,14 @@ for d in $(ls /dev/nvme?n1); do
     echo "Disk ${d} already mounted, skipping..."
   fi
 done
+for d in $(ls /dev/xvdd); do
```

Comment:
I think you can do something like `for d in $(ls /dev/{nvme?n1,xvdd})` and combine the two loops into one. If you want that to be slightly more explicit: `for d in $(ls /dev/nvume?n1 /dev/xvdd)`.